CODE_OF_CONDUCT.md

```diff
@@ -62,6 +62,7 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the maintainers responsible for enforcement at:
 
 alexvonme at matrix dot org
+(This is not an email address, you may reach it after opening an account in matrix dot org)
```

Comment:
```suggestion
(This is not an email address, you may reach it after opening an account on [matrix.org](https://matrix.org))
```
It's not an email a not soil spam, but a website. Let's people click on it.

Reply:
I actually commented on this, days before, then accidentally kept it at pending, see below for my latest comment.